README.md

```diff
@@ -21,6 +21,10 @@ The question type can be multiple-choice or single-choices.
 
```

Comment:
> The old quiz recap showed questions in the results you incorrectly answered or needed multiple attempts to get them right.

What confused me is that now if you answer the question correctly on the last attempt we show a `3` but if you answer it wrongly there would also be a `3`. I went with a "-" to avoid this. This is probably different from the picture you posted, where questions answered on the last attempt have a "2", questions answered on the second attempt have a "1" and questions answered correctly the first time would have a "0" but they don't appear. As we now want to show these questions, the attempts number for these is "1", so it all increases by one. 

Anyway, it's probably not that confusing for the user, but in any case that's something that will change from the old recap.

Reply:
As discussed, the old recap works the same way. I'll change "-" to "3" for wrongly answered questions.

Reply:
> 1. Questions I already tried to answer still had the selection from before if it is the last remaining question in the pool. Can we reset this so no pre-selection is done? It could confuse users and tempt to select the wrong answer again.
> 2. Questions where I used all three attempts show "Attempts -" in the results table. Shouldn't this be indicated with "Attempts 3"?

1. This should be fixed here:  https://github.com/christophblessing/quiz-recap/pull/39/commits/1f236844729149f5128893dd72618f1366e027cd
2. This one here: https://github.com/christophblessing/quiz-recap/pull/39/commits/04ff14d146700b6d6eb09d26db299e3421fcf005